graphql/image_test.go

```diff
@@ -21,6 +21,30 @@ func init() {
 	testutil.Setup()
 }
 
+func TestOperatingSystem(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestOperatingSystem")
```

Comment:
mega nit: you could just use `t.Name()` to avoid having to copy-paste the test name (I also think this is a dead/unused parameter, but that's a separate issue 🤷)

Reply:
omg yay... I'll update all of these!